easybuild/easyconfigs/p/PyMetis/PyMetis-2011.1.1-goolf-1.4.10-Python-2.7.3.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "PythonPackage"
+
+name = "PyMetis"
+version = "2011.1.1"
+
+homepage = "https://pypi.python.org/pypi/PyMetis"
+description = """PyMetis is a Python wrapper for the Metis graph partititioning software by George Karypis, Vipin Kumar and others."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://pypi.python.org/packages/source/P/%(name)s/']
+
+python = "Python"
+pythonversion = "2.7.3"
+versionsuffix = "-%s-%s" % (python, pythonversion)
+##versionsuffix = '-Python-2.7.3'
+
+metis = "METIS"
+metisversion = "5.0.2"
+
+dependencies = [
+    ('Boost', '1.51.0', versionsuffix),
+##    (python, pythonversion),
+#    (metis, metisversion), # OOPS, WHAT IS THE PROBLEM OVER HERE?
```

Comment:
Do not recall the original issue, so I will remove the hashes, so that this becomes directly testable with `--from-pr`.
(have I told you how much I like that feature? it makes eb to have a special place in my digital world ;-)